src/Mobs/Wolf.cpp

```diff
@@ -402,3 +402,37 @@ void cWolf::InStateIdle(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 }
 
 
+
+
+
+void cWolf::WriteMetadata(cMetadataWriter & a_Writer) const
+{
+	super::WriteMetadata(a_Writer);
+
+	// Ageable fields (since wolf inherits cPassiveAgressiveMonster instead of cPassiveMonster, this isn't handled directly)
+	a_Writer.WriteBool(IsBaby());
```

Comment:
This is a more obscure hierarchy problem. When you change `cPassiveMonster`, you would think, that everything is ok, since there are x classes inheriting and calling the changed method.

**How should you know about the Wolfes?**